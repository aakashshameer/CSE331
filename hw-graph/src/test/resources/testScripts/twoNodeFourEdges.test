#Test to create two nodes, two self-edges and two edges between different nodes

#Create a graph
CreateGraph dgraph

#Add two nodes
Added dgraph b
Added dgraph c

#Add two self edges
AddEdge dgraph b b BB
AddEdge dgraph c c CC

#Add two edges between different nodes
AddEdge dgraph b c BC
AddEdge dgraph c b CB

#Print the nodes in the graph
ListNodes dgraph

#Print outgoing edges from b
ListChildren dgraph b

#Print outgoing edges from c
ListChildren dgraph c