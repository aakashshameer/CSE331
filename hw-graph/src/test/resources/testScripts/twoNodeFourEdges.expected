#Test to create two nodes, two self-edges and two edges between different nodes

#Create a graph
create graph dgraph

#Add two nodes
added node b to dgraph
added node c to dgraph

#Add two self edges
added edge BB from b to b in dgraph
added edge CC from c to c in dgraph

#Add two edges between different nodes
added edge BC from b to c in dgraph
added edge CB from c to b in dgraph

#Print the nodes in the graph
dgraph contains: b c

#Print outgoing edges from b
the children of b in dgraph are: b(BB) c(BC)

#Print outgoing edges from c
the children of c in dgraph are: b(CB) c(CC)